tap_outreach/sync.py

```diff
@@ -227,9 +227,11 @@ def process_records(stream, mdata, max_modified, records, filter_field, fks):
                             raise Exception(
                                 'null or `id` field expected for `data` relationship')
 
-                        if fk_field_name in record_flat:
-                            raise Exception(
-                                '`{}` exists as both an attribute and generated relationship name'.format(fk_field_name))
+                        # January 2023: Let relationship value of record take precedence over attribute value (will take precedence at line 214)
+                        # (https://github.com/singer-io/tap-outreach/issues/20)
+                        # if fk_field_name in record_flat:
+                        #     raise Exception(
+                        #         '`{}` exists as both an attribute and generated relationship name'.format(fk_field_name))
```

Comment:
@seanlally I see the issue specifically for the prospects table as mentioned in the description. Should we consider bypassing the condition for the prospects table?

```suggestion
if stream != "prospects":
     if fk_field_name in record_flat:
          raise Exception(
                                '`{}` exists as both an attribute and generated relationship
                                name'.format(fk_field_name))
```